src/lib/ServoOutput/ServoMgr.cpp

```diff
@@ -29,15 +70,34 @@ void ServoMgr::writeMicroseconds(uint8_t ch, uint16_t valueUs)
 {
     const uint8_t pin = _pins[ch];
     if (pin == PIN_DISCONNECTED)
+    {
         return;
+    }
     _activePwmChannels |= (1 << pin);
 #if defined(PLATFORM_ESP32)
-    ledcWrite(ch, map(valueUs, 0, _refreshInterval[ch], 0, 65535));
+    ledcWrite(_chnMap[ch], map(valueUs, 0, _refreshInterval[ch], 0, (1 << _resolution_bits[ch]) - 1));
 #else
     startWaveform8266(pin, valueUs, _refreshInterval[ch] - valueUs);
 #endif
 }
 
+void ServoMgr::writeDuty(uint8_t ch, uint16_t duty)
+{
+    const uint8_t pin = _pins[ch];
+    if (pin == PIN_DISCONNECTED)
+    {
+        return;
+    }
+    duty = duty > 1000 ? 0 : duty; // prevent wraping around
```

Comment:
`duty = duty > 1000 ? 1000 : duty; `

Should be 1000?

Reply:
Oh, this line is fixing the below overflow from being seen on the scope!

Reply:
yes, this is a trick to do some protecting when wraping around because of any reason.